conf/messages.en

```diff
@@ -808,42 +796,40 @@ title.niue=Niue Island
 title.norfolk_island=Norfolk Island
 title.northern_mariana_islands=Northern Mariana Islands
 title.norway=Norway (including Svalbard Archipelago and Jan Mayen Island)
+title.north_macedonia=North Macedonia
 title.occupied_palestinian_territory=Occupied Palestinian Territory (West Bank (including East Jerusalem) and Gaza Strip)
 title.oman=Oman (formerly Muscat and Oman)
 title.pakistan=Pakistan
-title.palra_island=Palra Island
 title.palau=Palau
+title.title.palmyra_atoll=Palmyra Atoll
 title.panama=Panama (including the former Canal Zone)
 title.papua_new_guinea=Papua New Guinea
 title.paraguay=Paraguay
 title.peru=Peru
 title.philippines=Philippines
-title.pitcairn=Pitcairn island
+title.pitcairn=Pitcairn,Henderson,Ducie and Oeno Islands
 title.poland=Poland
 title.portugal=Portugal (including Azores and Madeira)
 title.puerto_rico=Puerto Rico
 title.qatar=Qatar
 title.ras_al_khaimah=Ras al Khaimah
 title.reunion=Reunion
 title.romania=Romania
-title.ross_dependency=Ross Dependency
 title.russia=Russia
 title.rwanda=Rwanda
 title.saba=Saba
-title.sabah=Sabah
 title.samoa=Samoa (formerly Western Samoa)
 title.saint_barthelemy=St Barthelemy
-title.st_eustatius=St Eustatius
+title.st_eustatius=Sint Eustatius
 title.saint_kitts_and_nevis=St Kitts and Nevis
-title.saint_helena_ascension_and_tristan_da_cunha=St Helena (including Ascension Island and Tristan da Cunha Islands)
+title.saint_helena_ascension_and_tristan_da_cunha=Saint Helena
 title.saint_lucia=St Lucia
-title.st_maarten_north=St Maarten (North)
-title.st_maarten_south=St Maarten (South)
+title.st_maarten_north=Saint Maarten (French Part)
```

Comment:
this is "Saint-Martin (French part)" in the spreadsheet

Reply:
Good spot, thanks! 👍